.github/workflows/wpiformat.yml

```diff
@@ -0,0 +1,34 @@
+name: Lint and Format
+
+on:
+  pull_request:
+  push:
```

Comment:
Let's change this to just `pull_request`. Also change the [`main.yml` workflow](https://github.com/SubZero-Robotics/2023-CommandRobot/blob/main/.github/workflows/main.yml) the same way. Place `branches: '**'` as well below `pull_request:`.